insights/combiners/limits_conf.py

```diff
@@ -0,0 +1,60 @@
+"""
+Limits configuration
+====================
+
+The limits files are normally available to rules as a list of LimitsConf
+objects.  This combiner turns those into one set of data, and provides a
+``find_all()`` method to search the rules from all the files.
+"""
+
+from insights.core.plugins import combiner
+from insights.parsers.limits_conf import LimitsConf
+
+
+@combiner(requires=[LimitsConf])
+class AllLimitsConf(object):
+    """
+    Combiner for accessing all the limits configuration files.
+
+    Attributes:
+        domains(set): the set of domains found in all data files.
+        limits(list): a list of the original LimitsConf parser results.
+        rules(list): the entire list of rules.
+    """
+    def __init__(self, local, shared):
+        rules = []
+        domains = set([])
+        self.limits = shared[LimitsConf]
+        for limits in shared[LimitsConf]:
+            # The rule dictionary contains the file_path as a key for later.
+            rules.extend(limits.rules)
+            domains.update(limits.domains)
+
+        self.rules = rules
+        self.domains = domains
+        super(AllLimitsConf, self).__init__()
+
+    def find_all(self, **kwargs):
```

Comment:
Would it be better for this combiner to provide an interface to return the `active/valid` setting? It seems the `limits.d/file` can override the same entry in `limits.conf`, https://access.redhat.com/solutions/199993.


just like what HttpdConfAll does?